report-ng/app/src/components/timings/test-timings/test-timings.scss

```diff
@@ -0,0 +1,26 @@
+/*!
+ * Testerra
+ *
+ * (C) 2023, Selina Natschke, Deutsche Telekom MMS GmbH, Deutsche Telekom AG
+ *
+ * Deutsche Telekom AG and all other contributors /
+ * copyright owners license this file to you under the Apache
+ * License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+.echart-timings {
+    width: 80em;
```

Comment:
Please do not define a fixed width. We should always use `width: 100%;`

Currently also the timings charts itself has fixed bars. The default echarts are flexible.